### DISCUSSION TITLE
How can i reset or overwrite the table class?

### DISCUSSION THREAD

**Discussion by woss** [saadeghi/daisyui#1706] [Q&A] [answered]
I need to style the table rows depending on the status. For example, some rows will be sponsored, some not, and for sponsored, I would like to have nice border animations and different backgrounds.

**Comment by saadeghi** [comment, marked as answer]
By default, row (`<tr>`) has no background but its children (`<th>` and `<td>`) inside it have a background color. If you want to set a background color to the row, you can first remove the children backgrounds using a class name like `[&>*]:bg-transparent` and then apply your own color with a class like `bg-secondary` or borders with border utility classes like `border-4 border-primary`.  
Example: https://play.tailwindcss.com/oFbcSP1RW6

> I would like to have nice border animations

There are many ways to add border animations and it depends on your design but you can basically do anything with the border here (like any other element) either with Tailwind CSS utility classes or if it's a really complex animation, using custom CSS.

Let me know if you have any questions.
